pkg/envoy/agent.go

```diff
@@ -105,13 +131,72 @@ func (a *Agent) terminate() {
 	if e != nil {
 		log.Warnf("Error in invoking drain listeners endpoint %v", e)
 	}
-	log.Infof("Graceful termination period is %v, starting...", a.terminationDrainDuration)
-	time.Sleep(a.terminationDrainDuration)
-	log.Infof("Graceful termination period complete, terminating remaining proxies.")
-	a.abortCh <- errAbort
+	time.Sleep(minDrainDuration)
```

Comment:
on L129 log what we are going to do maybe. Like `Agent draining proxy for 5s, then waiting for active connections to terminate...`